### PR TITLE
Add inventory script for ansible-playbook

### PR DESCRIPTION
**Pull request by keedya**
- Allow ansible to get hosts inventory from rackHD by node id
- the ansible job looks for this script in /opt/onrack/etc/ansible/

### CODE REVIEW

**Comment by keedya** [comment]
@RackHD/corecommitters  @zyoung51 @tannoa2 @uppalk1

**Comment by jlongever** [comment]
+1

**Comment by zyoung51** [comment]
+1

**Comment by benbp** [comment]
Do we also need to remove inventory.py?

File: scripts/rackhd.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+RACKHD_URL = 'http://localhost:8080'
```
Comment (benbp) [inline scripts/rackhd.py]: We might want to render this port as a template variable given that it's configurable by the user.

File: scripts/rackhd.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            node_info = json.loads(info['lookup'])
+            ipaddress = ''
+            if len(node_info) > 0:
+                ipaddress = node_info[0]["ipAddress"]
+            output += '  "' + nodeid + '" : {\n'
+            output += '    "hosts": [ "' + ipaddress + '" ],\n'
+            output += '    "vars" : {\n'
+            for key,result in info.iteritems():
+                output += '      "' + key + '": ' + json.dumps(json.loads(result), sort_keys=True, indent=2) + ',\n'
+            output += '      "ansible_ssh_user": "renasar"\n'
```
Comment (benbp) [inline scripts/rackhd.py]: I don't think any of our images still "renasar" as the user anymore, rather it's been changed to "monorail"

**Comment by benbp** [comment]
Two minor things that I _think_ might be a -1, but correct me if I'm wrong on either one.

File: scripts/rackhd.py (hunk 2 above)
Comment (keedya) [inline scripts/rackhd.py]: I did modify inventory.py in ansible job to rackhd.py

**Comment by benbp** [comment]
Ah ok @keedya, sorry I thought inventory.py as a file still existed somewhere, but looks like it doesn't. :+1:

**Pull request merged**